javascript/javascript3/week3/script.js

```diff
@@ -0,0 +1,285 @@
+//use class way original data
+let c = document.getElementById("myCanvas");
+c.width = window.innerWidth;
+c.height = window.innerHeight;
+let ctx = c.getContext("2d");
```

Comment:
const

Reply:
Hi dear Ida.is that mean we would better use const all the time?

Reply:
Exactly, always use const until it doesn't let you.

Reply:
okay, i remember now.Thanks.